src/wallet/coin_selection.rs

```diff
@@ -1507,16 +1508,19 @@ mod test {
             .map(|u| OutputGroup::new(u, fee_rate))
             .collect();
 
-        let size_of_change = 31;
-        let cost_of_change: u64 = (size_of_change as f32 * fee_rate.as_sat_vb()) as u64;
+        let change_out_size = 31_f32;
+        let change_in_size = 68_f32;
+        let cost_of_change: u64 = (change_out_size * fee_rate.as_sat_vb()
+            - change_in_size * LONG_TERM_FEE_RATE.as_sat_vb())
```

Comment:
I think the sign here is wrong. The two costs should add (unless one of them is tracked as negative for some reason).

 ```suggestion
             + change_in_size * LONG_TERM_FEE_RATE.as_sat_vb())
```